### ISSUE TITLE
alt+shift+f formats html but not css

### ISSUE THREAD

**Issue by myusrn** [Microsoft/vscode#59418]
Issue Type: <b>Bug</b>

I really like alt+shift+f keyboard shortcut for formatting html document content.  It appears this is not enabled for css document content which i woiuld expect it to do both for educational and easy of css format maintenance in the case of skilled users.

VS Code version: Code 1.27.2 (f46c4c469d6e6d8c46f268d1553c5dc4b475840f, 2018-09-12T16:17:45.060Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz (8 x 2112)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.85GB (6.30GB free)|
|Process Argv|C:\Users\ob1\AppData\Local\Programs\Microsoft VS Code\Code.exe|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (5)</summary>

Extension|Author (truncated)|Version
---|---|---
vbscript|Dar|1.0.4
nightswitch-lite|gha|2.1.0
csharp|ms-|1.16.1
vsliveshare|ms-|0.3.735
team|ms-|1.136.0

</details>

**Comment by aeschli** [comment]
Check out the marketplace for a CSS formatter.

The request to add a css formatter to the built-in css support is https://github.com/Microsoft/vscode/issues/19166
(but we actually rather let the use have a choice)

**Comment by myusrn** [comment]
Thanks for the response on this.  Yes I saw marketplace offerings, e.g. [ this 4 star ranked css formatter](https://marketplace.visualstudio.com/items?itemName=aeschli.vscode-css-formatter), but it doesn't hook up to the out of the box html document format keyboard shortcut alt+shift+f.  I opened [this ](https://github.com/aeschli/vscode-css-formatter/issues/9) issue in that extension's repo to see if there is a solution to that.

Not understanding why there isn't an oob offering that folks can use marketplace to get alternative versions of.  Just not following thinking that an oob offering with keyboard shortcut should exist for html5 but not dependent css3 documents.
